.github/workflows/autobuild.yml

```diff
@@ -168,6 +169,7 @@ jobs:
         uses:                       actions/checkout@v2
         with:
           submodules:               true
+          fetch-depth:              ${{ matrix.config.checkout_fetch_depth || '1' }}
```

Comment:
Does the `checkout_fetch_depth: '0'` further up evaluate here as false or true?

Reply:
I haven't found explicit documentation, but the CI confirms that it does what it should: [Android uses a full checkout](https://github.com/jamulussoftware/jamulus/runs/5267232585?check_suite_focus=true#step:3:40) while the other targets (e.g. [Linux](https://github.com/jamulussoftware/jamulus/runs/5267232613?check_suite_focus=true#step:3:40)) only fetch the most recent commit:

Therefore, I assume that `'0'` as a non-empty string evaluates to `true` (probably like Python).

Reply:
> Therefore, I assume that `'0'` as a non-empty string evaluates to `true` (probably like Python).

And unlike Perl and PHP!